agent/base

```diff
@@ -244,10 +244,19 @@ function verify_common_bench_script_options {
 export INVENTORY=/tmp/inventory.$$
 trap "rm -f $INVENTORY" QUIT INT EXIT
 function generate_inventory {
```

Comment:
Seems like we should promote `generate_inventory` to a separate script that lives in `util-scripts`, perhaps ... or ... leverage the new Perl Ansible module that @atheurer recently did instead.

Having the trap in the base is not great because it fires whether or not a caller invoked generate_inventory or not.